types.d.ts

```diff
@@ -0,0 +1,12 @@
+interface PageInfo {
+  pageName: string;
+  pageLink: string;
+  pageTitle: string;
+  displayInNav: boolean;
+}
+// Yaml Config
+// TODO: Make this only match the config.yaml file
```

Comment:
isn't all we have to do for that to replace the wildcard operator with the word config? If so, fix it up

Reply:
No I tried that but it was giving me trouble. I think it's Todo with oath resolution of some sort.

Reply:
ok, worked on my machine it seemed but don't want to risk a change that would break the site